pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -95,6 +101,111 @@ func (r *ReconcileMasterUserRecord) Reconcile(request reconcile.Request) (reconc
 		return reconcile.Result{}, err
 	}
 
-	// Do nothing for now
+	for _, account := range userRecord.Spec.UserAccounts {
+		err = r.ensureUserAccount(reqLogger, account, userRecord)
+		if err != nil {
+			reqLogger.Error(err, "unable to synchronize with member UserAccount")
+			return reconcile.Result{}, err
+		}
+	}
+
 	return reconcile.Result{}, nil
 }
+
+func (r *ReconcileMasterUserRecord) ensureUserAccount(log logr.Logger, recAccount toolchainv1alpha1.UserAccountEmbedded, record *toolchainv1alpha1.MasterUserRecord) error {
+	// get & check fed cluster
+	fedCluster, err := r.getMemberCluster(log, recAccount, record)
```

Comment:
`fedCluster` and `getMemberCluster`; wrt naming use either `fed` / `member` for consistency :-)

Reply:
renamed: 53e9ca6